### PR TITLE
VOA-2065

### PR DESCRIPTION
**Pull request by valydia**
# After
Screenshot 2020-07-14 at 15 51 36

Based on the prototype: https://voa-bar-prototype.herokuapp.com/ 
user: voa / p/w: hmrc

### CODE REVIEW

File: app/controllers/CouncilTaxStartController.scala [hunk 1]
```diff
@@ -1,66 +0,0 @@
-/*
```
Comment (valydia) [inline app/controllers/CouncilTaxStartController.scala:1]: This page is being skipped now - not needed anymore

File: app/views/councilTaxStart.scala.html [hunk 2]
```diff
@@ -1,49 +0,0 @@
-@*
```
Comment (valydia) [inline app/views/councilTaxStart.scala.html:1]: This page is being skipped now - not needed anymore

**Comment by russellthorn** [comment]
My only comment is design related - I'm not sure there's a need to hide the bulleted list inside a details reveal/dropdown. I worked on a service a long time ago for called secure file upload service and we simply laid the criteria for uploads in a list above the upload form input.

**Comment by valydia** [comment]
Thanks for your comment, what you described seems to be regarded the details tag, seems to be like the old design. I ll bring that up in my next catch up with Dan!

**Pull request merged**